### PR TITLE
created workflow to copy adapter components to docs in meshery

### PR DESCRIPTION
**Pull request by asubedy**
Signed-off-by: asubedy <frexpe@pm.me>

**Description**

This PR fixes #

**Notes for Reviewers**
These are some caveats I discovered till now using this approach
1. Suppose we want to place our .md files on `docs/_components/linkerd` folder then at least `docs/_components` folder should be present on the Meshery repo
~2. No skip rule added, (I could not figure out how to add it) so any PR may run this workflow.~

P.S: No unnecessary commits are taking place while the workflow runs

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@Revolyssup

File: .github/workflows/components-to-doc.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        uses: crykn/copy_folder_to_another_repo_action@v1.0.6
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.GH_ACCESS_TOKEN }}
+        with:
+          source_folder: 'output'
+          destination_repo: 'meshery/meshery'
+          destination_folder: 'docs/_integrations/linkerd'
+          destination_branch: 'master'
+          user_email: 'frexpe@pm.me'
+          user_name: 'frexpe'
```
Comment (alphaX86) [inline .github/workflows/components-to-doc.yml:49]: Isn't this field to be set as l5io?

Comment (adithyaakrishna) [reply .github/workflows/components-to-doc.yml:49]: Yes, @asubedy Could you please have a look at this?

**Comment by asubedy** [comment]
Data will be represented in the following way
image

File: .github/workflows/components-to-doc.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            
+            function join_by {
+              local d=${1-} f=${2-}
+              if shift 2; then
+                echo "$f${@/#/$d}"
+                
+              fi
+            }
+            mkdir output
+            folder_path=templates/oam/workloads/*
```
Comment (leecalcote) [inline .github/workflows/components-to-doc.yml:29]: @Revolyssup when this filepath changes... let's be sure to include a global find and replace. // @gr455

**Review by leecalcote — APPROVED** [review]
Let's give this a try...

**Comment by leecalcote** [comment]
@asubedy [this workflow](https://github.com/meshery/meshery-linkerd/actions/runs/2792271456) should probably be embedded within or trigger from the completion of components being generated, not triggered on merge to master.

**Pull request merged**